measures/SimulationOutputReport/measure.rb

```diff
@@ -37,8 +37,8 @@ def arguments
 
     # make an argument for including optional end use subcategories
     arg = OpenStudio::Measure::OSArgument::makeBoolArgument("include_enduse_subcategories", true)
```

Comment:
Maybe change argument name to "disaggregate_interior_equipment" ?

Reply:
I intentionally left it so that no changes to buildstockbatch would need to be made. Perhaps though we shouldn't take the easy road.